src/ServicePulse.Host/app/js/services/service.prettify.js

```diff
@@ -0,0 +1,40 @@
+; (function (window, angular, undefined) {
+    'use strict';
+
+    function service($jquery) {
+        
+        this.prettifyText = function (text) {
+            if ((typeof text === 'object' && text.constructor === Object) || isJSON(text) ) {
+                return vkbeautify.json(text);
```

Comment:
Angular has a built in json filter

`<div>{{jsonvalue | json}}</div>`

So maybe we just need a library for xml?

Reply:
Perhaps https://github.com/krtnio/angular-pretty-xml?

Reply:
That one seems to only do XML.

Reply:
Angular has a built in filter for json, why do we have to include a separate one?>

Reply:
https://docs.angularjs.org/api/ng/filter/json

Reply:
Didn't know, even better.

Reply:
I am fine with that